### ISSUE TITLE
Journeymap Integration. 

### ISSUE THREAD

**Issue by mysticdrew** [AHilyard/MerchantMarkers#6]
I am the Journeymap Dev, I looked through your integration. Thanks for adding it. 
However, all of what you did with the mixins can be done with the EntityRadarUpdateEvent through our API. 
https://github.com/TeamJM/journeymap-api/blob/1.16.5_1.8/src/main/java/journeymap/client/api/event/forge/EntityRadarUpdateEvent.java
examplemod:
https://github.com/TeamJM/journeymap-api/blob/5de6fda29c72992f1b806bb56c7475c727edd84a/src/main/java/example/mod/client/plugin/ForgeEventListener.java#L192

ExampleCode with how Pixelmon updates entity textures:

```java
    @SubscribeEvent
    public void onRadarEntityUpdateEvent(EntityRadarUpdateEvent event) {
        LivingEntity entity = (LivingEntity)event.getWrappedEntity().getEntityLivingRef().get();
        if (entity instanceof PixelmonEntity) {
            PixelmonEntity pixelmon = (PixelmonEntity)entity;
            event.getWrappedEntity().setEntityIconLocation(pixelmon.getSprite());
        }
    }

**Comment by mysticdrew** [comment]
Feel free to pop into our discord to chat about integration. I'd much rather devs use our API than mixins to avoid breaking things. I can add more hooks if needed, thanks!

**Comment by AHilyard** [comment]
This was fixed by #7, thank you!
